### PR TITLE
Work with SSM via wrapper

### PR DESCRIPTION
**Pull request by igorkotof**
Based on https://github.com/isabey-cogni/ssm-wrapper:

- Added list functionality
- Added push functionality
- delete functionality should be fixed via PR to the opensource project (I've already checked the code - it should be easy)

Testing:
[asciicast](https://asciinema.org/a/epX0oq1YCOsxgHGH62zRAHkAq)

### CODE REVIEW

**Comment by igorkotof** [comment]
[Implementation ssm-tool for parameter store](https://trello.com/c/cFFUhCkJ/204-implementation-ssm-tool-for-parameter-store)

**Review by AutomationD — COMMENTED** [review]
Do we really need to have a separate command for this? I'm curious.

**Comment by igorkotof** [comment]
> Do we really need to have a separate command for this? I'm curious.

@AutomationD no, it's just to show and for testing - if you like it - I will update with deleting old command

**Comment by AutomationD** [comment]
> > Do we really need to have a separate command for this? I'm curious.
> 
> @AutomationD no, it's just to show and for testing - if you like it - I will update with deleting old command

Ah, makes sense. 

@igorkotof could you please give more overview of which tool are we using, maybe post a link in the PR text?

**Comment by igorkotof** [comment]
> > > Do we really need to have a separate command for this? I'm curious.
> > 
> > 
> > @AutomationD no, it's just to show and for testing - if you like it - I will update with deleting old command
> 
> Ah, makes sense.
> 
> @igorkotof could you please give more overview of which tool are we using, maybe post a link in the PR text?

@AutomationD I've added

**Comment by AutomationD** [comment]
> > Do we really need to have a separate command for this? I'm curious.
> 
> @AutomationD no, it's just to show and for testing - if you like it - I will update with deleting old command

@igorkotof So, what's the plan with global secrets?

**Comment by igorkotof** [comment]
> > > Do we really need to have a separate command for this? I'm curious.
> > 
> > 
> > @AutomationD no, it's just to show and for testing - if you like it - I will update with deleting old command
> 
> @igorkotof So, what's the plan with global secrets?

@AutomationD keep like $(INFRA_DIR)/env/$(ENV)/secrets/global.json

File: aws/secrets.mk [hunk 1]
```diff
@@ -1,36 +1,47 @@
 # Macroses
 ########################################################################################################################
+# SSM Wrapper
+SSM_DOCKER_TAG ?= hazelops/ssm-wrapper:latest
+SSM ?= $(DOCKER) run -v $(HOME)/.aws/:/root/.aws -v $(ENV_DIR):/$(ENV_DIR) -e AWS_PROFILE=$(AWS_PROFILE) -e AWS_REGION=$(AWS_REGION) $(SSM_DOCKER_TAG) ssm
```
[the remark this answers is not in the document]
Comment (igorkotof) [reply aws/secrets.mk]: @gloovers yes, updated - please take a look and resolve conversation

File: aws/secrets.mk [hunk 2]
```diff
@@ -1,36 +1,47 @@
 # Macroses
 ########################################################################################################################
+# SSM Wrapper
+SSM_DOCKER_TAG ?= hazelops/ssm-wrapper:latest
```
Comment (AutomationD) [inline aws/secrets.mk:4]: Let's rename this to be more descriptive. Wouldn't `SSM_WRAPPER_DOCKER_IMAGE` or something similar be more logical? Or am I misunderstanding something?

**Review by AutomationD — CHANGES_REQUESTED** [review]
LGTM,

File: aws/secrets.mk (hunk 2 above)
Comment (igorkotof) [reply aws/secrets.mk:4]: it makes sense

Comment (igorkotof) [reply aws/secrets.mk]: done

**Review by AutomationD — APPROVED** [review]
Let's ensure we've done final tests with all current changes.
Otherwise it looks good to me, thank you!

**Pull request merged**